tree.js

```diff
@@ -0,0 +1,37 @@
+// (function test(){
+// console.log('test')
+// })();
+function toText(arr) {
+  const recur = ({ name, items }) =>
```

Comment:
maybe could be implemented in a way to not having an internal recur function (in theory it might create a new recur function on every toText call, but that's really a micro optimization, so just saying things loud)